modules/pipeline/pipengine/actionexecutor/plugins/scheduler/executor/plugins/k8sflink/flink.go

```diff
@@ -200,6 +201,10 @@ func (k *K8sFlink) BatchDelete(ctx context.Context, tasks []*spec.PipelineTask)
 	return nil, nil
 }
 
+func (k *K8sFlink) Inspect(ctx context.Context, task *spec.PipelineTask) (apistructs.TaskInspect, error) {
+	return apistructs.TaskInspect{}, errors.New("k8sflink don`t support inspect")
```

Comment:
doesn't

Reply:
`Inspect` will be invoked internally only, so no error needed?

Reply:
return error means 'taskop wait' doesn`t need to report task inspect, becase k8sflink and k8sspark don`t support inspect